proxy/http2/Http2SessionAccept.cc

```diff
@@ -54,9 +54,8 @@ Http2SessionAccept::accept(NetVConnection *netvc, MIOBuffer *iobuf, IOBufferRead
   }
 
   Http2ClientSession *new_session = THREAD_ALLOC_INIT(http2ClientSessionAllocator, this_ethread());
-  new (new_session) Http2ClientSession();
-  new_session->acl            = std::move(session_acl);
-  new_session->accept_options = &options;
+  new_session->acl                = std::move(session_acl);
+  new_session->accept_options     = &options;
```

Comment:
Why the extra spaces?

Reply:
It seems like line 58 only needs one space before the = .  Looks like a bug in clang-format then.

Reply:
I mean line 56 by the previous line.